.github/workflows/rust.yml

```
@@ -81,7 +81,7 @@ jobs:
       - run: rustup target add wasm32-unknown-unknown
       # TODO: we should probably manage the container in the test itself
       - run: make build-test-wasms
-      - run: docker run -d -p 8000:8000 stellar/quickstart:soroban-dev --standalone --enable-soroban-rpc --enable-core-artificially-accelerate-time-for-testing --protocol-version 20
+      - run: docker run -d -p 8000:8000 stellar/quickstart:pr-389-soroban-dev --standalone --enable-soroban-rpc --enable-core-artificially-accelerate-time-for-testing --protocol-version 20
```

We don't need to merge this, as e2e tests don't block PRs. Thanks for checking that the updated image works though. 
```suggestion
      - run: docker run -d -p 8000:8000 stellar/quickstart:soroban-dev --standalone --enable-soroban-rpc --enable-core-artificially-accelerate-time-for-testing --protocol-version 20
```